src/lib/viewers/BaseViewer.js

```diff
@@ -1097,15 +1098,18 @@ class BaseViewer extends EventEmitter {
         );
         this.annotationControls.setMode(nextMode);
 
-        if (this.options.enableAnnotationsDiscoverability) {
-            switch (nextMode) {
-                case AnnotationMode.REGION:
-                    this.containerEl.classList.add(CLASS_ANNOTATIONS_CREATE_REGION);
-                    break;
-                default:
-                    this.containerEl.classList.remove(CLASS_ANNOTATIONS_CREATE_REGION);
-                    break;
-            }
+        [CLASS_ANNOTATIONS_CREATE_HIGHLIGHT, CLASS_ANNOTATIONS_CREATE_REGION].forEach(createClass =>
+            this.containerEl.classList.remove(createClass),
+        );
+        switch (nextMode) {
+            case AnnotationMode.HIGHLIGHT:
+                this.containerEl.classList.add(CLASS_ANNOTATIONS_CREATE_HIGHLIGHT);
+                break;
+            case AnnotationMode.REGION:
+                this.containerEl.classList.add(CLASS_ANNOTATIONS_CREATE_REGION);
+                break;
+            default:
+                break;
```

Comment:
You may need to resolve conflicts because another just merged PR moved these logic into a new function.